packages/orm/prisma/src/generator/domain/DmmfModel.ts

```
@@ -67,9 +67,9 @@ export class DmmfModel {
   }
 
   addImportDeclaration(moduleSpecifier: string, name: string, isTypeOnly = false) {
-    if (!this.#imports.has(moduleSpecifier)) {
-      moduleSpecifier = resolveExtension(moduleSpecifier);
+    moduleSpecifier = resolveExtension(moduleSpecifier);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)